curriculum/challenges/spanish/06-information-security-and-quality-assurance/information-security-and-quality-assurance-projects/anonymous-message-board.spanish.md

```diff
@@ -59,4 +59,5 @@ tests:
 ```js
 // solution required
```

Comment:
Grammar Error: 

 Cree una aplicación de JavaScript de pila completa que sea funcionalmente similar a esto
**+  "Cree una aplicación de JavaScript de pila completa que sea funcionalmente similar a esta"**

Reply:
Thanks for the review comment @SamuelSHernandez. 

However, the translation is not a part of the changes in this PR per say. If you are interested, you are welcome to make a separate PR :)

I am going ahead and marking this comment resolved.